demos/code/code.js

```diff
@@ -272,6 +274,24 @@ Code.tabClick = function(clickedName) {
     }
   }
 
+  if (document.getElementById('tab_json').classList.contains('tabon')) {
+    var jsonTextarea = document.getElementById('content_json');
+    var jsonText = jsonTextarea.value;
+    var json = null;
+    try {
+      json = JSON.parse(jsonText);
+    } catch (e) {
+      var q = window.confirm(MSG['badJson'].replace('%1', e));
```

Comment:
@NeilFraser How do I add a new message key? Or modify the existing one to not be XML specific?

Reply:
The demos/code/msg/xx.js files are static and not connected with any translation service.  I'd be inclined to edit the files to rename "xmlError" to "loadError", rename "badXml" to "parseError", replace "XML" with "%1", and inject "XML" or "JSON" into the "%1" when using that message.  Use your IDE's search and replace for this, since Arabic and Hebrew is dangerous to edit correctly.

Those are just my suggestions.  Of course use your best judgement.

Reply:
I think that plan worked! Here's the old arabic vs the new arabic:
![Screenshot 2021-09-09 08 07 30](https://user-images.githubusercontent.com/25440652/132712561-3d63c777-0ae8-4a26-bedb-e14b7dc6adbc.png)
![Screenshot 2021-09-09 08 07 55](https://user-images.githubusercontent.com/25440652/132712570-8715daf8-6cb0-41f0-9dad-e5b8ad51dbf6.png)